### PR TITLE
feat(analytics): Add tracking param to sentry issue links

### PR DESCRIPTION
**Pull request by adhiraj**
We have no idea where people are coming from when they view a sentry issue. This adds a referrer=X query param to issue links we post on external properties.

Will require some changes to plugins repo as well, doing it in a bit.

### CODE REVIEW

File: src/sentry/models/group.py [hunk 1]
```diff
@@ -262,10 +262,11 @@ def save(self, *args, **kwargs):
         )
         super(Group, self).save(*args, **kwargs)
 
-    def get_absolute_url(self):
-        return absolute_uri(
-            reverse('sentry-group', args=[self.organization.slug, self.project.slug, self.id])
-        )
+    def get_absolute_url(self, referrer=None):
```
Comment (EvanPurkhiser) [inline src/sentry/models/group.py]: Should this maybe come in as kwargs for extra query string parameters?

Comment (adhiraj) [reply src/sentry/models/group.py]: sure can do

File: tests/sentry/integrations/jira/test_integration.py [hunk 2]
```diff
@@ -406,7 +406,10 @@ def get_client():
                             'string_max_length=self.string_max_length)\n\nmessage\n{code}'
                             ) % (
                                 group.qualified_short_id,
-                                absolute_uri(group.get_absolute_url()),
+                                absolute_uri(
+                                    group.get_absolute_url(
+                                        params={
+                                            'referrer': 'jira_integration'})),
```
Comment (EvanPurkhiser) [inline tests/sentry/integrations/jira/test_integration.py]: Urgh, this got so ugly, is this what autopep8 does?

**Review by evanpurkhiser — APPROVED** [review]

File: tests/sentry/integrations/jira/test_integration.py (hunk 2 above)
Comment (adhiraj) [reply tests/sentry/integrations/jira/test_integration.py]: yeah it's auto. i think it's nice :P

**Pull request merged**